users/static/users/scripts/captcha-refresh.js

```diff
@@ -0,0 +1,9 @@
+$(function () {
+    $("#captcha-refresh").click(function()  {
+        $.getJSON("/captcha/refresh/", {}, function(json) {
+            document.getElementsByClassName('captcha')[0].attributes.src.value = json.image_url;
+            document.getElementById("id_captcha_0").value = json.key;
+        });
+        return false;
+    });
+});
```

Comment:
Was this the source of that major frustration of yours that you manifested so verbally last Friday? 😆

Reply:
@kbeker You are mixing jQuery style with vanilla Javascript style here, works for me but if you want you could unify this :)

Reply:
@maciejSamerdak exactly that was it. It didn't want to cooperate with me last friday :P @rwrzesien when I was doing this it was driving me crazy so when it finally started to work I just left it and did not touch :D But I'll try to unify this :)

Reply:
@rwrzesien oh, now it did work from first shot. Changed completely to `jQuery`.